index.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Github Notifier</title>
+    <title>GitHub Notifier</title>
```

Comment:
Can you change this to `Gitify` instead of `GitHub Notifier`? I guess I didn't update it when I decided on the app name.

Thanks for all the other changes!